stock_view_adj_hls/views/stock_quant_views.xml

```diff
@@ -25,4 +25,16 @@
             </xpath>
         </field>
     </record>
+
+    <record id="view_stock_move_line_operation_tree" model="ir.ui.view">
```

Comment:
Please create a new file `stock_move_line_views.xml` and put this record in there.

Ref. https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#file-naming

Reply:
Add new file.

And edit [description.rst.](https://github.com/qrtl/hls-custom/pull/141/files#diff-856934fdd42372f4d9ada7c9abdaa7fdec5fd6846d2ce5e334be3f246e7b810bR1)